Northstar.Custom/mod/scripts/vscripts/gamemodes/_gamemode_inf.gnut

```diff
@@ -44,6 +44,9 @@ void function SelectFirstInfectedDelayed()
 {
 	wait 10.0 + RandomFloat( 5.0 )
 
+	if ( file.hasHadFirstInfection ) // in case a player joins and gets infected before this happens
+		return
+

```

Comment:
Wouldn't it make more sense to make anyone joining before first infected to spawn as survivor instead? ^^

Reply:
Yeah, that's possible too, either way it would achieve the same result I guess.

Reply:
Yeah, personally I'd go for the that option then ^^

Reply:
Resolved with 81631c612f6b0c602d44f1b1701dd5fee31778ca